Templates/Databases/MicrosoftSQL/Query Microsoft SQL Database and return JSON GET/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Query Microsoft SQL Database and return JSON GET",
```

Comment:
Lets rename the process to something more findable. Like "HTTP endpoint returning Microsoft SQL data as JSON".

Reply:
Renamed to the name you suggested.